frontend/lib/js/index.js

```diff
@@ -4,24 +4,14 @@ import 'babel-polyfill';
 
 import './fixEdge';
 
-import React                      from 'react';
-import ReactDOM                   from 'react-dom';
-import {
-  compose,
-  applyMiddleware,
-  createStore,
-}                                 from 'redux';
-import { Provider }               from 'react-redux';
-import createHistory              from 'history/createBrowserHistory';
+import React                           from 'react';
+import ReactDOM                        from 'react-dom';
+import { AppContainer as HotReloader } from 'react-hot-loader';
 
 import './localization'; // To initialize locales
 import './app/actions'; //  To initialize parameterized actions
-
-import { BASENAME, isProduction } from './environment';
-import createMiddleware           from './middleware';
-
-import AppRouter                  from './app/AppRouter';
-import reducers                   from './app/reducers';
+import {store, browserHistory }        from './store'
```

Comment:
Could you add some spaces between these curly braces?
Also, we typically use single quotes for includes, let's keep it consistent.